src/stages/stage6_events.c

```diff
@@ -458,16 +460,15 @@ void elly_newton(Boss *b, int t) {
 		}
 
 		PROJECTILE(
-			.sprite_ptr = apple,
-			.pos = clamp(creal(global.plr.pos) + nfrand() * 64, apple->w*0.5, VIEWPORT_W - apple->w*0.5),
```

Comment:
this is the one you already found i suppose?

Reply:
Indeed. It's unfortunate that we shipped v1.2 with this.